book/content/open_research/02/softwarelicenses.md

```diff
@@ -1,5 +1,7 @@
 # Software licenses
 
+*Disclaimer: Good legal advice is timely, specific, and given by an expert; this chapter is none of these. It was written by an engineer, not by a lawyer, and it's a heavily simplified overview of a very complex field. The intent is to give you an overview of the basics, so that you will know when to check whether something you want to do has potential legal ramifications. Don't make any important decisions based solely on the contents of this chapter.*
```

Comment:
Please move disclaimers to the end of the chapter or rephrase it to be added right after the next two statements defining the license.